sql/catalyst/src/main/java/org/apache/spark/sql/connector/write/BatchWrite.java

```diff
@@ -45,8 +45,10 @@
    *
    * If this method fails (by throwing an exception), the action will fail and no Spark job will be
    * submitted.
+   *
+   * @param info Physical information about the input data that will be written to this data writer.
```

Comment:
`will be written to this data writer.`

I'd say `will be written to this table`. We write one partition to each data writer, but the physical information is not of one partition, but the entire dataset.